cmd/spire-server/cli/run/run.go

```diff
@@ -64,23 +64,24 @@ type Config struct {
 }
 
 type serverConfig struct {
-	BindAddress    string             `hcl:"bind_address"`
-	BindPort       int                `hcl:"bind_port"`
-	CAKeyType      string             `hcl:"ca_key_type"`
-	CASubject      *caSubjectConfig   `hcl:"ca_subject"`
-	CATTL          string             `hcl:"ca_ttl"`
-	DataDir        string             `hcl:"data_dir"`
-	DefaultSVIDTTL string             `hcl:"default_svid_ttl"`
-	Experimental   experimentalConfig `hcl:"experimental"`
-	Federation     *federationConfig  `hcl:"federation"`
-	JWTIssuer      string             `hcl:"jwt_issuer"`
-	JWTKeyType     string             `hcl:"jwt_key_type"`
-	LogFile        string             `hcl:"log_file"`
-	LogLevel       string             `hcl:"log_level"`
-	LogFormat      string             `hcl:"log_format"`
-	RateLimit      rateLimitConfig    `hcl:"ratelimit"`
-	SocketPath     string             `hcl:"socket_path"`
-	TrustDomain    string             `hcl:"trust_domain"`
+	AuditLogEnabled bool               `hcl:"auditlog_enabled"`
```

Comment:
I think I would prefer to see this named `audit_log_enabled` since "audit log" is two words